packages/pwa-buildpack/lib/WebpackTools/configureWebpack.js

```diff
@@ -52,7 +52,13 @@ function getMode(cliEnv = {}, projectConfig) {
     return 'development';
 }
 
-async function configureWebpack({ context, vendor = [], special = {}, env }) {
+async function configureWebpack({
+    context,
+    vendor = [],
+    special = {},
+    env,
+    enableServiceWorkerDebugging
```

Comment:
Made a similar comment in another place, but I thought this was already part of the `env`?